### DISCUSSION TITLE
Any way to customize the Miller title content in a plotPDF()

### DISCUSSION THREAD

**Discussion by zmichels** [mtex-toolbox/mtex#803] [Ask Anything] [answered]
I have been using MTEX to analyze geological field data. This first plot is just an example of some field data (black squares are foliation poles, red circles are lineation directions, great circles are the trace of the foliation plane):

![workshop_geo_field_example_Lin_ _Fol_legend](https://user-images.githubusercontent.com/8769619/108760145-e4989c80-7512-11eb-9c71-b3bb453c64b3.png)

One reason I prefer to use MTEX is that I can combine foliation and lineation measurements into single orientation/rotation representations of the field fabric. After doing that, I have a set of orientations for which certain miller directions correspond to the original fabric elements. In this example case, the lineations are represented by [100] and the poles to foliation are represented by [010]. This is useful for computing means, modes, etc.,  from a number of measurements while ensuring that the result maintains the fabric relationship between the lineation and foliation (i.e., perpendicular). I say it is useful because the mean directions of the different fabric elements (foliation poles and lineations) separately do not necessarily maintain the perpendicular characteristic of fabrics from which they were computed. Whereas they do when computing from a fabric **orientation**.

Anywhoo, I like using the plotPDF() command to easily image the different axes and their densities on the same scale, and annotate with other things, etc. EXCEPT... I really want to be able to modify the Miller names in the plot. I am not sure if this can be done through some handles of the axes or something, or could there be a 'DisplayName' when defining Millers? This next plot show the same foliation and lineation data plotted as mentioned, and you see the Miller names above each.

![workshop_field_example_fabPDF](https://user-images.githubusercontent.com/8769619/108761444-7b198d80-7514-11eb-8c67-553d09ef6133.png)

Any thoughts or suggestions are greatly appreciated.

EDIT: To be clear... I would like to rename the axes in the plot to something like "foliation" and "lineation"

Cheers,
Zach

**Comment by zmichels** [comment, marked as answer]
Welp, I guess I solved my own question right after asking it. I suppose someone else may find it useful some day.

Here's how I would do it for a two-Miller polefigure I guess (it works for the example above):

`% modify the "Miller" labels to reflect the plot`
`f = gcf;`
`f.Children(end).Title.String = 'foliation';`
`f.Children(end-1).Title.String = 'lineation';`

Cheers,
Zach

![workshop_field_example_fabPDF_newAxNames](https://user-images.githubusercontent.com/8769619/108764299-08121600-7518-11eb-8a62-1066d91241dc.png)

**Reply by JinkuiMeng** [reply]
Dear Zach,

I want to know how to add a ruler when ploting, such as the above pole figure

<img width="408" alt="plot mud" src="https://user-images.githubusercontent.com/64880536/108818974-2732a580-75f5-11eb-9046-0a2f318db79f.png">

best,

Jinkui

**Reply by bright-men** [reply]
Dear zmichels,
I'd like to ask you how to hide the xy coordinates on the pole figure.
best wishes.

**Comment by lfgmorales** [comment]
You mean the scale of your contour plots?


that can be done with mtexColorbar after your plotPDF command


https://mtex-toolbox.github.io/Annotations.html



cheers



L


Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
________________________________

**Reply by JinkuiMeng** [reply]
Thank you!

**Comment by lfgmorales** [comment]
Hi

Write some like this on the begining of scrpit will do it

setMTEXpref(‚pfAnnotations‘,@(varargin) 1);

Cheers

L
Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 29 Jan 2024, at 04:02, bright-men ***@***.***> wrote:

﻿

Dear zmichels,
I'd like to ask you how to hide the xy coordinates on the pole figure.
best wishes.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/803#discussioncomment-8276366>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6XDFPCU72PQLWXRHTYQ4GMDAVCNFSM4YBETEUKU5DIOJSWCZC7NNSXTOKENFZWG5LTONUW63SDN5WW2ZLOOQ5TQMRXGYZTMNQ>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/803#discussioncomment-8276366", "url": "https://github.com/mtex-toolbox/mtex/discussions/803#discussioncomment-8276366", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by bright-men** [reply]
Many thanks!
